usaspending_api/search/v2/views/spending_by_category.py

```diff
@@ -236,10 +275,13 @@ def recipient(self) -> list:
 
         self.queryset = self.common_db_query(filters, values)
         # DB hit here
-        query_results = list(self.queryset[self.lower_limit : self.upper_limit])
+        query_results = list(self.queryset[self.lower_limit: self.upper_limit])
```

Comment:
When I run black it includes spaces around the semicolon operator

Reply:
Weird... when I run BLACK it removes them.  It also upsets my linter as not being PEP8 compliant.

Reply:
black is pep8 compliant.... your linter isn't

Reply:
https://www.python.org/dev/peps/pep-0008/#pet-peeves

```
Avoid extraneous whitespace in the following situations:

- Immediately before a comma, semicolon, or colon:
    Yes: if x == 4: print x, y; x, y = y, x
    No:  if x == 4 : print x , y ; x , y = y , x
```

I think you need to give your BLACK a tuneup.  :-p

Reply:
Did you even read the page?

```However, in a slice the colon acts like a binary operator, and should have equal amounts on either side (treating it as the operator with the lowest priority). In an extended slice, both colons must have the same amount of spacing applied. Exception: when a slice parameter is omitted, the space is omitted.```

![image](https://user-images.githubusercontent.com/22781949/64459723-d30ccc00-d0b5-11e9-83a2-9d1b63db762d.png)

Reply:
https://github.com/psf/black#slices

Reply:
Clearly not far enough.

Reply:
Although this was interesting:

> This behaviour may raise E203 whitespace before ':' warnings in style guide enforcement tools like Flake8. Since E203 is not PEP 8 compliant, you should tell Flake8 to ignore these warnings.